Gemfile

```diff
@@ -4,3 +4,5 @@ source "https://rubygems.org"
 
 # Specify your gem's dependencies in lamian.gemspec
 gemspec
+
+gem "rubocop-config-umbrellio", github: "umbrellio/code-style"
```

Comment:
Why do you use github version?

Reply:
I am sure you have an access to do it

Reply:
Nope.

Reply:
Well, i definitely not want to refer github HEAD of any repo, beacause there is no lock and when HEAD change travis can start failing without any changes in repo. I think it is better not to update this to HEAD than to update in this case.

If you want to update please at least specify commit hash